gbpservice/nfp/configurator/drivers/firewall/asav/asav_fw_driver.py

```diff
@@ -237,6 +237,42 @@ def _get_device_interface_name(self, cidr):
 
         return 'interface-' + cidr.replace('/', '_')
 
+    def _log_forwarding(self, mgmt_ip_address):
+
+        configuration_data = {
+            "protocol": "UDP",
+            "ip": {
+                "kind": "IPv4Address",
+            },
+            "secureEnabled": False,
+            "interface": {
+                "kind": "objectRef#Interface",
+                "name": "management"
+            },
+            "emblemEnabled": False
+        }
+
+        configuration_data['ip']['value'] = self.conf.log_forward_ip_address
+        configuration_data['port'] = self.conf.log_forward_port
+
+        url = const.REQUEST_URL % (mgmt_ip_address,
+                                   "/api/logging/syslogserver")
+
+        status = self.rest_api.post(url, configuration_data, self.auth)
+        if not status:
+            return False
+
+        url = const.REQUEST_URL % (mgmt_ip_address, "/api/cli")
+
+        enable_trap_data = {
+            "commands": [
+                "logging trap informational"
+            ]
+        }
+        status = self.rest_api.post(url, enable_trap_data, self.auth)
```

Comment:
If the return value of this function is True/False, then check the status and return accordingly.